compose/dm-store.yml

```diff
@@ -36,8 +35,8 @@ services:
       SPRING_DATASOURCE_URL: jdbc:postgresql://ccd-shared-database:5432/evidence
       SPRING_DATASOURCE_USERNAME: "${DB_USERNAME}"
       SPRING_DATASOURCE_PASSWORD: "${DB_PASSWORD}"
-      #ENABLE_AZURE_STORAGE_CONTAINER: "false"
-      #ENABLE_POSTGRES_BLOB_STORAGE: "true"
+      ENABLE_AZURE_STORAGE_CONTAINER: "true"
+      ENABLE_POSTGRES_BLOB_STORAGE: "false"
```

Comment:
wouldn't be better for local development to stay on postgres?

Reply:
we use azure storage emulator for local development simulating what production uses, so its safe to turn postgress off.